### ISSUE TITLE
Make a sticky Navbar

### ISSUE THREAD

**Issue by Mayuresh-22** [Mayuresh-22/Minglr-Social-Network-Site#1]
### What's the Issue?
Currently, when a user scrolls down the navbar gets hidden and the user needs to scroll to the top to access the navbar again.
Therefore, this issue needs to be solved.
Thank You

**Comment by ShatilKhan** [comment]
Hi , I'd like to take on this issue.
Please assign it to me

**Comment by Mayuresh-22** [comment]
Hey @ShatilKhan , I will assign this issue to you.

**Comment by ankit-369** [comment]
hey @ShatilKhan i would like to work on this issue can you assign me

**Comment by Mayuresh-22** [comment]
Any updates? @ShatilKhan

**Comment by ShatilKhan** [comment]
working on it @Mayuresh-22 
I'll make my PR today

**Comment by AbdullahTheDev** [comment]
If it is not done, can you assign this to me under hacktoberfest label?

**Comment by Mayuresh-22** [comment]
Yes @AbdullahTheDev you can start working on it I'll assign it to you.

**Comment by AbdullahTheDev** [comment]
I completed this task, but unfortunately I am unable to make a pull request, can you please accept my last pull request?

**Comment by Mayuresh-22** [comment]
Okay @AbdullahTheDev I will look into it

**Comment by 4darsh-Dev** [comment]
hey, @Mayuresh-22 if task is not completed can I make it

**Comment by AbdullahTheDev** [comment]
> hey, @Mayuresh-22 if task is not completed can I make it 

Task is completed
